tips/06-08-2017/Readme.md

```diff
@@ -1,14 +1,11 @@
-#### Subscribe to our [Telegram channel](https://t.me/dailytip) to get daily updates  
+# live templates
+useful shortcuts for WebStorm [live templates feature](https://www.jetbrains.com/help/webstorm/live-templates-2.html)
 
----
+<p align="center">
```

Comment:
Setting [width=700] will create inconveniences for mobile readers. This should be:
```![live template demo](http://res.cloudinary.com/dzsjwgjii/image/upload/v1502207798/rcc-demo.gif)```